src/averageAges.js

```diff
@@ -9,17 +9,28 @@
  * To calculate century:
  * Divide year of person's death by 100: Math.ceil(person.died / 100)
  *
- * @param {object[]} people
- * @param {number} century - optional
+ // * @param {object[]} people
+ // * @param {number} century - optional
  *
  * @return {number}
  */
+
+function averageAge(peopleArray) {
+  return (peopleArray.reduce((sum, x) =>
+    sum + x.died - x.born, 0)) / peopleArray.length;
```

Comment:
this break looks weird
```suggestion
  return peopleArray.reduce(
    (sum, x) => sum + x.died - x.born,
    0,
  ) / peopleArray.length;
```